<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>date_set</class>
 <widget class="QDialog" name="date_set">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Выбор периода</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>448</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_from">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd MMMM yyyy</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_to">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>166</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd MMMM yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>125</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>С</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>По</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Один день</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Период</string>
   </property>
  </widget>
  <widget class="QComboBox" name="templateBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Вчера</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сегодня</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Завтра</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Прошлая неделя</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Текущая неделя</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Следующая неделя</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Прошлый месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Текущий месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Следующий месяц</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>date_from</tabstop>
  <tabstop>date_to</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>calendarWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>date_set</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>date_set</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>date_to</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>794</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>793</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>date_to</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>817</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
